tensorflow/core/kernels/mkl_tfconv_op.cc

```diff
@@ -29,8 +29,8 @@ limitations under the License.
 #include "tensorflow/core/util/tensor_format.h"
 
 #include "tensorflow/core/util/mkl_util.h"
-#include "third_party/mkl/include/mkl_dnn.h"
-#include "third_party/mkl/include/mkl_dnn_types.h"
+#include "mkl_dnn.h"
```

Comment:
In Google projects, always use full include path for C++/ObjC programs if you can. That's the convention. 
If you have no other options, there's a couple of things you can do
- guard the full path inclusion for google use (PLATFORM_GOOGLE), and partial path for others. 
- in addition, you should use <lib.h> rather than "lib.h" whenever possible.